### ISSUE TITLE
First charcter of custom url is missing from link

### ISSUE THREAD

**Issue by TransformReality** [YOURLS/YOURLS#2316]
### Technical details regarding my environment

* YOURLS version: v 1.7.2
  * Plugins enabled: reCaptcha, Don't Log Bots, Anti spam

### Reproducible bug summary 

The current behavior is when a user makes a custom url, it removes the first letter of their custom url. Example: The user chooses a custom shortlink of "short.co/test", but after clicking "Shorten" the newly made link reads "short.co/est". it removes the first character.

Not sure why its doing this now, it was working fine before. I haven't altered any functions.
### Reproducible bug summary 

Not sure how to replicate it, it just appeared. To recap when a user trys to make a custom link it remove the first character of whatever custom url they put in and makes that a link instead.

**Comment by ozh** [comment]
It most likely removes a char that's not allowed by the defined charset, eg "Test" -> "est".

**Comment by TransformReality** [comment]
Thata what originally through but I checked them and that's not.it; it does not matter what the character is it will remove the first character every time.

**Comment by ozh** [comment]
You should continue investigate to pinpoint the precise conditions that make you encounter this. Nowhere in YOURLS code there is something related to which device is used.

**Comment by TransformReality** [comment]
Okay I'll look to see if I can find the cause, something I noticed is it doesn't happen on mobile with Firefox, but it does occur on Chrome and Opera browser.

**Comment by fscheps** [comment]
Hi there, I´ve just installed YOURLS through Softaculous installed (February 2019) version 1.7.3 and I am facing exactly the same issue. 
I actually find Yourls behavior quite erratic, I tried to edit links and wont let me giving me messages like the URL already exists. 
Or Trying to create short links for URL´s like Yahoo and such as tests and not allowing me to do so.

I recorded a video with the reported first character being removed behavior: 

https://www.dropbox.com/s/nmihh9xheun1amg/YourlsStrangeBehavior.mp4?dl=0
